### ISSUE TITLE
what's this project current status?

### ISSUE THREAD

**Issue by bung87** [tulayang/nimnode#1]
hi 
could you provide more documents, about features ,examples, tutorious, also information about development that anyone interested can contribute to.

**Comment by tulayang** [comment]
@bung87

This project is experimental and no longer developed. I am not sure when to write a document. Here is a simple guide:

- uv - the libuv API in nim (map to C API) [see libuv documentation](http://docs.libuv.org/en/v1.x/)

- error - ``NodeError`` extends ``Exception`` used for reportting errors

- streams - the basic abstraction of read and write streams. For reading, there are mainly some listening events, such as reading a piece of data (``"data"`` event), end of data (``"end"`` event), error (``"error"`` event). For writing, an internal write queue used to hold asynchronous write operations.

- net - TCP client and server based on streams

- http - HTTP client and server based on net

**Comment by bung87** [comment]
ok good to know,I think you can write these to readme and using GitHub Pages hosting api docs, and let this repo archived if you really dont want continue developing this project, I guess you already write some example codes when you developed this project, I'd like to see it in readme.

**Comment by tulayang** [comment]
@bung87 

You can see the [unit tests](https://github.com/tulayang/nimnode/tree/master/test) for now.

**Comment by bung87** [comment]
yeah ,I'v read these tests, I'm expecting a `test_http.nim` there.
